src/test/java/com/rcv/TabulatorTests.java

```diff
@@ -81,7 +81,8 @@ private static boolean fileCompare(String path1, String path2) {
           break;
         }
         // both files have content so compare it
-        if (!line1.equals(line2)) {
+        if (!(line1.contains("GeneratedDate") && line2.contains("GeneratedDate"))
```

Comment:
This is pretty hacky. The alternative would be to have a dummy timestamp that we always use for test data, and alter ResultsWriter so that it knows to populate that dummy timestamp instead of the real timestamp when it's operating in a test environment. That seems less hacky, I guess...?

Reply:
Agreed.  I made an issue for it so we can defer.